src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -390,10 +386,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
 #Region "Buffer helpers"
 
+        <Runtime.CompilerServices.MethodImpl(Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)>
```

Comment:
Why are we adding these attributes?  I don't think we should be using these unless we have a profile which proves their necessity.